vars/runATH.groovy

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env groovy
+
+/**
+ * Wrapper for running the ATH see README and runATH.txt for full documentation
+ */
+
+def call(Map params = [:]) {
+    def athUrl = params.get('athUrl', 'https://github.com/jenkinsci/acceptance-test-harness.git')
+    def athRevision = params.get('athRevision', 'master')
+    def metadataFile = params.get('metadataFile', 'essentials.yml')
+    def jenkins = params.get('jenkins', 'latest')
```

Comment:
Possibly rename this parameter to include some version in it? jenkins alone seems generic enough :)

Reply:
Keep in mind that you can pass a war file, so if I include `version` it will be misleading, also keeping the same format than the local scripts here.

I am open to discuss that, but that should be done in a separate PR which should also change the ATH local scripts, not here